packages/ui/src/utls/lib.ts

```diff
@@ -0,0 +1,6 @@
+import { clsx, type ClassValue } from "clsx";
```

Comment:
From [shadcn](https://ui.shadcn.com/docs/installation/manual). This helper makes it easier to conditionally add Tailwind CSS classes.